### ISSUE TITLE
BrowserNav no longer detects existing bookmarks in NVDA 64-bit alpha builds

### ISSUE THREAD

**Issue by amirsol81** [mltony/nvda-browser-nav#33]
@mltony I know it might be a bit early to ask, but I wanted to report a compatibility issue with the 64-bit alpha versions of NVDA. BrowserNav mostly works fine, but the bookmark navigation feature no longer functions correctly.
When pressing J or Shift + J to move between previously created bookmarks, BrowserNav cannot find them and reports a message as if no bookmarks have been created for the document. Other parts of the add-on appear to work normally.
It would be great if this could be reviewed for compatibility with the new 64-bit alpha builds of NVDA, since users of these builds cannot currently use bookmark navigation. Many of us prefer to stay on alpha builds as much as possible to help with early testing and feedback.
Thank you very much for your time and for maintaining this incredibly useful add-on.

**Comment by mltony** [comment]
I plan to work on 64-bit compatibility around Christmas/New year time.

**Comment by mltony** [comment]
I just published release 2.7.7 here. Should be compatible with latest alpha versions. Can you check that it works for you?

**Comment by amirsol81** [comment]
@mltony Thanks for the awesome job! It works flawlessly with the latest NVDA 64-Bit alpha, and I can use all bookmarks. Yet again, your efforts are much appreciated.

**Comment by amirsol81** [comment]
@mltony I opened an issue #35 about BrowserNav 27.7 and its inability to create bookmarks for headings and links. I tested this on 2 Windows 11 Pro machines, running NVDA 64.Bit alphas.
